code/modules/resleeving/machines.dm

```diff
@@ -529,6 +529,7 @@
 		var/obj/item/device/nif/nif = new MR.nif_path(occupant,MR.nif_durability)
 		for(var/path in MR.nif_software)
 			new path(nif)
+		nif.durability = MR.nif_durability //Restore backed up durability as the line above forces reinstall wear for each soft.
```

Comment:
You may want to remove the MR.nif_durability on 529 (test to make sure that works though, maybe it's used as an indicator we're being resleeved and is somehow important, I don't remember)

The reason: What if someone has 5 durability on their NIF in the backup?
- Create NIF with 5 durability
- Install seve-- oh no we broke it, installs may have failed
- Does setting durability back up even work now? Who knows.

Reply:
Good catch. Seems the durability bit for the new nif isn't used for anything other than setting the durability on spawn so it shouldn't be breaking anything